packages/components/src/border-box-control/types.ts

```diff
@@ -44,12 +44,11 @@ export type BorderBoxControlProps = ColorProps &
 		 */
 		value: AnyBorder;
 		/**
-		 * Start opting into the larger default height that will become the
-		 * default size in a future version.
+		 * Size of the control.
 		 *
-		 * @default false
+		 * @default default
```

Comment:
Same as above for all the `'default'` values and the README updating in this file